tests/functional/notifier.py

```diff
@@ -94,6 +92,9 @@ def create_verification_issue(chart_name, chart_owners, report_url, software_nam
     access_token -- An optional github access token secret. If not passed will try to get from GITHUB_AUTH_TOKEN environment variable\n
     """
 
+    if os.environ.get('NOTIFY_ID'):
```

Comment:
Use this `NOTIFY_ID` if provided to override the chart_owners to tag

Reply:
The notify id could be "OWNERS", "none" or a githib id. I would expect to see some logic depending in its value, like reading the OWNERS file if set to OWNERS. Also if "none" there is no id to notify.